packages/@okta/vuepress-site/docs/reference/rl-system-log-events/index.md

```diff
@@ -0,0 +1,267 @@
+
+```json
+{
+  "actor": {
+    "id": "00uw8nGF9OiREtZyr0g3",
+    "type": "User",
+    "alternateId": "oie-u1@oie.okta1.com",
+    "displayName": "oie1 oie1",
+    "detailEntry": null
+  },
+  "client": {
+    "userAgent": {
+      "rawUserAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:83.0) Gecko/20100101 Firefox/83.0",
+      "os": "Mac OS X",
+      "browser": "FIREFOX"
+    },
+    "zone": "null",
+    "device": "Computer",
+    "id": null,
+    "ipAddress": "127.0.0.1",
+    "geographicalContext": {
+      "city": null,
+      "state": null,
+      "country": null,
+      "postalCode": null,
+      "geolocation": {
+        "lat": 37.74,
+        "lon": -122.39999999999999
+      }
+    }
+  },
+  "authenticationContext": {
+    "authenticationProvider": null,
+    "credentialProvider": null,
+    "credentialType": null,
+    "issuer": null,
+    "interface": null,
+    "authenticationStep": 0,
+    "externalSessionId": "1025poeucCTQVK22GxJEK1Y-g"
+  },
+  "displayMessage": "Operation rate limit violation",
+  "eventType": "system.operation.rate_limit.violation",
+  "outcome": {
+    "result": "DENY",
+    "reason": "Too many enrollment attempts for SMS factor"
+  },
+  "published": "2020-12-10T04:46:12.033Z",
+  "securityContext": {
+    "asNumber": null,
+    "asOrg": null,
+    "isp": null,
+    "domain": null,
+    "isProxy": null
+  },
+  "severity": "WARN",
+  "debugContext": {
+    "debugData": {
+      "operationRateLimitSubtype": "SMS",
+      "operationRateLimitTimeUnit": "MINUTES",
+      "operationRateLimitScopeType": "user",
+      "operationRateLimitSecondsToReset": "99",
+      "operationRateLimitThreshold": "15",
+      "operationRateLimitTimeSpan": "5",
+      "requestUri": "/api/v1/authn/factors",
+      "url": "/api/v1/authn/factors?updatePhone=true",
+      "phoneNumber": "+14037894561",
+      "authnRequestId": "reqWXOTNi2FQV6sUFQxWGCf8A",
+      "countryCallingCode": "1",
+      "requestId": "reqS9xgtpvOTcukX8Yu-SLRDQ",
+      "threatSuspected": "false",
+      "operationRateLimitType": "phone_enrollment"
+    }
+  },
+  "legacyEventType": null,
+  "transaction": {
+    "type": "WEB",
+    "id": "reqS9xgtpvOTcukX8Yu-SLRDQ",
+    "detail": {}
+  },
+  "uuid": "a0b60b8a-3aa2-11eb-8d69-abfc0c06b0f7",
+  "version": "0",
+  "request": {
+    "ipChain": [
+      {
+        "ip": "127.0.0.1",
+        "geographicalContext": null,
+        "version": "V4",
+        "source": null
+      }
+    ]
+  },
+  "target": [
+    {
+      "id": "SMS",
+      "type": "Factor Type",
+      "alternateId": null,
+      "displayName": null,
+      "detailEntry": null
+    },
+    {
+      "id": "/api/v1/authn/factors",
+      "type": "URL Pattern",
+      "alternateId": null,
+      "displayName": null,
+      "detailEntry": null
+    }
+  ]
+}
+```
+
+The following is an example System Log rate limit event where too many OTP verification attempts were made for the Email factor.
```

Comment:
@shayekhazaeli-okta do we want this example included?

Reply:
yes, lets keep this example, but make a note that this is only available for IDX users/flows